test/ruby/test_compile_prism.rb

```diff
@@ -303,6 +303,27 @@ def test_XStringNode
       # RUBY
     end
 
+    ############################################################################
+    # Structures                                                               #
+    ############################################################################
+
+    def test_ArrayNode
+      test_prism_eval("[]")
+      test_prism_eval("[1, 2, 3]")
+      test_prism_eval("%i[foo bar baz]")
+      test_prism_eval("%w[foo bar baz]")
```

Comment:
Can we add a test case for a non-static array too?

Reply:
The `%w` is non-static because the strings aren't frozen by default